games-emulation/pcsx2/metadata.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<herd>proxy-maintainers</herd>
+	<maintainer>
+		<email>ykonotopov@gmail.com</email>
+		<name>Yuri Konotopov</name>
+	</maintainer>
+	<use>
+		<flag name="png">Allow to dump texture as PNG (require <pkg>dev-cpp/pngpp</pkg>)</flag>
+		<flag name="wxwidgets3">Link against version 3.0 of <pkg>x11-libs/wxGTK</pkg> instead of 2.8</flag>
```

Comment:
No, no, no, that's not how we handle wxwidgets. Your previous version was better. I suggested trying to check if wxwidgets.eclass has some code to find out the best wxwidgets version, not adding USE flags. In fact, if that proves too hard, you can use 3.0 only since there's the long-time goal of switching to new versions as soon as they're supported.